### PR TITLE
Upgrading versions

### PR DESCRIPTION
**Pull request by krmahadevan**
The below user facing artifacts have been upgraded

* snakeyaml
* jquery webjars

### CODE REVIEW

File: versions.properties [hunk 1]
```diff
@@ -94,15 +134,31 @@ version.org.spockframework..spock-core=2.1-groovy-3.0
...
+##                         # available=2.2-M2-groovy-4.0
+##                         # available=2.2-M3-groovy-2.5
+##                         # available=2.2-M3-groovy-3.0
+##                         # available=2.2-M3-groovy-4.0
+##                         # available=2.3-groovy-2.5
+##                         # available=2.3-groovy-3.0
+##                         # available=2.3-groovy-4.0
+
+version.org.webjars..jquery=3.6.1
+##              # available=3.6.1
```
Comment (juherr) [inline versions.properties:154]: Remove this line has you updated

File: versions.properties [hunk 2]
```diff
@@ -94,15 +134,31 @@ version.org.spockframework..spock-core=2.1-groovy-3.0
...
 
 version.org.xmlunit..xmlunit-assertj=2.9.0
 
-version.org.yaml..snakeyaml=1.30
+version.org.yaml..snakeyaml=1.33
+##              # available=1.31
+##              # available=1.32
+##              # available=1.33
```
Comment (juherr) [inline versions.properties:161]: Remove this 3 lines as you updated

**Review by juherr — APPROVED** [review]

File: versions.properties (hunk 2 above)
Comment (krmahadevan) [reply versions.properties]: done

File: versions.properties (hunk 1 above)
Comment (krmahadevan) [reply versions.properties]: done

**Pull request merged**